Sample Data/ASIM/CommonSecurityLog_Schema.csv

```diff
@@ -0,0 +1,161 @@
+FileModificationTime,48,"System.String",string
+FilePath,49,"System.String",string
+FilePermission,50,"System.String",string
+FileType,51,"System.String",string
+FileName,52,"System.String",string
+FileSize,53,"System.Int32",int
+ReceivedBytes,54,"System.Int64",long
+Message,55,"System.String",string
+OldFileCreateTime,56,"System.String",string
+OldFileHash,57,"System.String",string
+OldFileID,58,"System.String",string
+OldFileModificationTime,59,"System.String",string
+OldFileName,60,"System.String",string
+OldFilePath,61,"System.String",string
+OldFilePermission,62,"System.String",string
+OldFileSize,63,"System.Int32",int
+OldFileType,64,"System.String",string
+SentBytes,65,"System.Int64",long
+EventOutcome,66,"System.String",string
+Protocol,67,"System.String",string
+Reason,68,"System.String",string
+RequestURL,69,"System.String",string
+RequestClientApplication,70,"System.String",string
+RequestContext,71,"System.String",string
+RequestCookies,72,"System.String",string
+RequestMethod,73,"System.String",string
+ReceiptTime,74,"System.String",string
+SourceHostName,75,"System.String",string
+SourceMACAddress,76,"System.String",string
+SourceNTDomain,77,"System.String",string
+SourceDnsDomain,78,"System.String",string
+SourceServiceName,79,"System.String",string
+SourceTranslatedAddress,80,"System.String",string
+SourceTranslatedPort,81,"System.Int32",int
+SourceProcessId,82,"System.Int32",int
+SourceUserPrivileges,83,"System.String",string
+SourceProcessName,84,"System.String",string
+SourcePort,85,"System.Int32",int
+SourceIP,86,"System.String",string
+StartTime,87,"System.DateTime",datetime
+SourceUserID,88,"System.String",string
+SourceUserName,89,"System.String",string
+EventType,90,"System.Int32",int
+DeviceEventCategory,91,"System.String",string
+DeviceCustomIPv6Address1,92,"System.String",string
+DeviceCustomIPv6Address1Label,93,"System.String",string
+DeviceCustomIPv6Address2,94,"System.String",string
+DeviceCustomIPv6Address2Label,95,"System.String",string
+DeviceCustomIPv6Address3,96,"System.String",string
+DeviceCustomIPv6Address3Label,97,"System.String",string
+DeviceCustomIPv6Address4,98,"System.String",string
+DeviceCustomIPv6Address4Label,99,"System.String",string
+DeviceCustomFloatingPoint1,100,"System.Double",real
+DeviceCustomFloatingPoint1Label,101,"System.String",string
+DeviceCustomFloatingPoint2,102,"System.Double",real
+DeviceCustomFloatingPoint2Label,103,"System.String",string
+DeviceCustomFloatingPoint3,104,"System.Double",real
+DeviceCustomFloatingPoint3Label,105,"System.String",string
+DeviceCustomFloatingPoint4,106,"System.Double",real
+DeviceCustomFloatingPoint4Label,107,"System.String",string
+DeviceCustomNumber1,108,"System.Int32",int
+FieldDeviceCustomNumber1,109,"System.Int64",long
+DeviceCustomNumber1Label,110,"System.String",string
+DeviceCustomNumber2,111,"System.Int32",int
+FieldDeviceCustomNumber2,112,"System.Int64",long
+DeviceCustomNumber2Label,113,"System.String",string
```

Comment:
what this field contain? any threat related info?

Reply:
DeviceCustomNumebr2Label is impact in case of Intrusion event but we do not have value available in DeviceCustomNumber2. Hene we haven't considered it for mapping.